src/display/api.js

```diff
@@ -1522,6 +1522,14 @@ var WorkerTransport = (function WorkerTransportClosure() {
     this.destroyCapability = null;
     this._passwordCapability = null;
 
+    this._networkStream = null;
+    this._fullReader = null;
+    if (pdfDataRangeTransport /* === true = chunkedViewerLoading */) {
```

Comment:
Nit: The comment isn't completely easy to read at first, and furthermore I'm not sure how much value it adds!?